scripts/scr_unit_size/scr_unit_size.gml

```diff
@@ -1,23 +1,26 @@
 
-function scr_unit_size(armour, role, other_factors) {
+function scr_unit_size(armour, role, other_factors, mobility) {
 
     // armour: armour
     // role: role
     // other_factors: +?
     var sz = 1;
     if (role!=""){
-
-        if (string_count("Dread",armour)>0) {sz=5;}
-        else if (role="Rhino") {sz=10;}
+		var bulky_armour = ["Terminator Armour", "Tartaros"]
+        if (string_count("Dread",armour)>0) {sz+=5;}else if (array_contains(bulky_armour,armour)){sz +=1};
+        if (role="Rhino") {sz=10;}
         else if (role="Predator") {sz=10;}
         else if (role="Land Raider") {sz=20;} 
         else if (role="Land Speeder") {sz=5;}
         else if (role="Whirlwind") {sz=10;}
         else if (role="Harlequin Troupe") {sz=5;}
+		else if (role="Chapter Master"){sz+=1;}
     
-        if (other_factors=false) then sz=sz*-1;
+        //if (other_factors=false) then sz=sz*-1;

```

Comment:
why commented out instead of removing? and also what does this do, invert the size?

Reply:
this script is a pre existing script and is used in many places, i've searched and could find no example where this line was relevant but left it just in case. also yeah just makes the size negative